package/MDAnalysis/lib/distances.py

```diff
@@ -504,10 +497,11 @@ def _determine_method(reference, configuration, max_cutoff, min_cutoff=None, box
                                configuration.max(axis=0)])
             size = max_dim.max(axis=0) - min_dim.min(axis=0)
         elif box is not None:
-            if box.shape[0] == 6:
+            if (box[3:] == 90).all():  # ortho
```

Comment:
Testing equality on float is a bad idea. `np.allclose` is more appropriate here.